### PR TITLE
Updated image-banner-overlay to v3.1.0

### PR DESCRIPTION
**Pull request by trevorjonesd2l**

### CODE REVIEW

**Comment by trevorjonesd2l** [comment]
It's not super clear to me why `d2l-simple-overlay` and `vui-input` were updated - I didn't update them in the `bower.json` and I don't believe they were released recently?

**Comment by dlockhart** [comment]
Try this:
- revert everything
- `bower-locker unlock`
- delete your `bower_components` directory
- update `image-banner-overlay` to the new version you want
- `bower install`
- `bower-locker lock`

File: bower.json [hunk 1]
```diff
@@ -1,7 +1,6 @@
 {
   "name": "brightspace-integration",
   "dependencies": {
-    "Stickyfill": "https://github.com/wilddeer/stickyfill.git#10d9853eda95659364a6932a03831a4df6252bf0",
```
Comment (dlockhart) [inline bower.json]: It looks like people on Windows that run `bower-locker` move this down alphabetically with the lowercase `s` and others have it appear first. I don't know how to fix this but it super sucks that it keeps moving around depending on who runs it, resulting in conflicts. :(

**Comment by trevorjonesd2l** [comment]
I actually did run those commands, specifically:

`bower-locker unlock`
`rm -rf bower_components`
Update bower.json: `"d2l-image-banner-overlay": "Brightspace/d2l-image-banner-overlay#^3.0.2",` -> `"d2l-image-banner-overlay": "Brightspace/d2l-image-banner-overlay#^3.1.0",`
`bower install`
`bower-locker lock`

File: bower.json [hunk 2]
```diff
@@ -35,7 +34,7 @@
     "d2l-rubric": "https://github.com/Brightspace/d2l-rubric.git#198e03f16de6aad87704774210c858144ea951b0",
     "d2l-scroll-spy": "https://github.com/BrightspaceUI/scroll-spy.git#1163917d64637971ac6d3c8e16ca31c1350c269d",
     "d2l-search-widget": "git://github.com/Brightspace/d2l-search-widget-ui.git#4829536ff872f8625ebce21d299356b182752edb",
-    "d2l-simple-overlay": "git://github.com/Brightspace/simple-overlay.git#4b6376b578344c2a3be58d3d17f8895ab86d32a8",
+    "d2l-simple-overlay": "git://github.com/Brightspace/simple-overlay.git#925242feab5f8bd5d7a3fa7da1cec6654ebbd122",
```
Comment (dlockhart) [inline bower.json]: I think your version here (`9252...`) is the correct one. `master` must be messed up.

File: bower.json [hunk 3]
```diff
@@ -68,21 +67,21 @@
...
+    "Stickyfill": "https://github.com/wilddeer/stickyfill.git#10d9853eda95659364a6932a03831a4df6252bf0",
     "susy": "https://github.com/ericam/susy.git#edcded39bf4626b072d7768b70b26a9dfee02d17",
     "vui-breadcrumbs": "https://github.com/Brightspace/valence-ui-breadcrumbs.git#1a0c1af6745a73b601597eb0d965df9c329bbc12",
     "vui-dropdown": "https://github.com/Brightspace/valence-ui-dropdown.git#5c50e4d8ae2d19b33514c477c20812f2264837f0",
     "vui-field": "https://github.com/Brightspace/valence-ui-field.git#613d3cc4fb370ac6ed020fecda4c64c089ed4610",
     "vui-focus": "https://github.com/Brightspace/valence-ui-focus.git#d092b7aa3c9eff70cbbbfc8567317f31fee4e563",
     "vui-gradient": "https://github.com/Brightspace/valence-ui-gradient.git#3c6a8ce532ffb298b7f303e5097e6a91c1d3f4e0",
     "vui-grid-system": "https://github.com/Brightspace/valence-ui-grid-system.git#0384e87cd068617419d42b3c8a8b6cb773fc507d",
-    "vui-input": "https://github.com/Brightspace/valence-ui-input.git#cbd467b877c06e081446531e97794fe2939a6d75",
+    "vui-input": "https://github.com/Brightspace/valence-ui-input.git#52a259d79389632bae31830c32d7303a5db14345",
```
Comment (dlockhart) [inline bower.json]: Same here -- yours is the correct version (`52a25..`)

**Pull request merged**